tests/test_vectorized_mobject.py

```diff
@@ -204,3 +207,33 @@ def test_vgroup_item_assignment_only_allows_vmobjects():
     vgroup = VGroup(VMobject())
     with pytest.raises(TypeError, match="All submobjects must be of type VMobject"):
         vgroup[0] = "invalid object"
+
+
+def test_bounded_become():
```

Comment:
It would be even better to have other tests testing the correct trimming of the subpaths, I think.